R/server.R

```diff
@@ -340,6 +340,14 @@ app_server <- function(input, output, session) {
   output$usage_plot <- renderPlotly({
     withProgress(message = "Plotting data ...", {
       unique_codes <- length(unique(filtered_data()$code))
+      
+      # When there are 500 or less selected codes, impute 0 usage 
+      # in the annual usage gaps
```

Comment:
```suggestion
      # When there are 500 or less selected codes, impute 0 usage 
      # in the annual usage gaps. We do this for the summary plot 
      # with all codes and the individual codes plot. We chose the
      # cutoff for this feature to be 500 because we expect this to
      # be useful in two main scenarios: (1) a user selects a small 
      # number of codes (or a codelist) and looks ad individual codes;
      # (2) a user selects only one or fiew codes looks at the summary 
      # plot, therefore we need to fill gaps also in the summary plot. 
```